cmd/controller/main.go

```diff
@@ -237,3 +245,33 @@ func newBuildpackRepository(keychain authn.Keychain) func(store *expv1alpha1.Sto
 		}
 	}
 }
+
+const controllerCount = 9
+
+//lifted from knative.dev/pkg/injection/sharedmain
+func genericControllerSetup(ctx context.Context, cfg *rest.Config) (*zap.SugaredLogger, *configmap.InformedWatcher, *http.Server) {
+	sharedmain.MemStatsOrDie(ctx)
+
+	// Adjust our client's rate limits based on the number of controllers we are running.
+	cfg.QPS = float32(controllerCount) * rest.DefaultQPS
+	cfg.Burst = controllerCount * rest.DefaultBurst
+	ctx, _ = injection.Default.SetupInformers(ctx, cfg)
+
+	logger, atomicLevel := sharedmain.SetupLoggerOrDie(ctx, component)
```

Comment:
lol @ number of methods named `DoThingOrDie`

Reply:
Knative loves going all in